app/tests/v2/test_users.py

```diff
@@ -421,3 +421,13 @@ def test_admin_status_no_data(self):
         self.assertEqual(data['status'], 400)
         self.assertEqual(data['error'], 'No data was provided')
         self.assertEqual(res.status_code, 400)
+
+    def test_list_users(self):
```

Comment:
You'd also want to create a new user and assert that `len(data['data'])` will increase by 1.